gobblin-metastore/src/main/java/org/apache/gobblin/metastore/MysqlStateStore.java

```diff
@@ -104,7 +104,7 @@
       "CREATE TABLE IF NOT EXISTS $TABLE$ (store_name varchar(100) CHARACTER SET latin1 not null,"
           + "table_name varchar(667) CHARACTER SET latin1 not null,"
           + " modified_time timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,"
-          + " state mediumblob, primary key(store_name, table_name))";
```

Comment:
This belongs to a previous commits:https://github.com/apache/incubator-gobblin/pull/2189  
Not sure why it shows up here.